phoenix-core/src/main/java/org/apache/phoenix/execute/AggregatePlan.java

```diff
@@ -112,7 +114,26 @@ private AggregatePlan(StatementContext context, FilterableStatement statement, T
     public Expression getHaving() {
         return having;
     }
-    
+
+    @Override
+    public Cost getCost() throws SQLException {
+        Long byteCount = getEstimatedBytesToScan();
+        if (byteCount == null) {
+            return Cost.ZERO;
+        }
+
```

Comment:
What does parallelLevel represent? Is that the level of parallelization done on the client? Is that derivable from thread pool size or cluster size? For now, maybe have a constant and doc it a bit.